lib/reference/functions.js

```diff
@@ -14,7 +14,28 @@ const camelCaseFunctions = new Set([
 	'skewY',
 ]);
 
-const mathFunctions = new Set(['calc', 'clamp', 'max', 'min']);
+const mathFunctions = new Set([
+	'abs',
+	'acos',
+	'asin',
+	'atan',
+	'atan2',
+	'calc',
+	'clamp',
+	'cos',
+	'exp',
+	'hypot',
+	'log',
+	'max',
+	'min',
```

Comment:
Could you add `minmax` and `sign`?
https://developer.mozilla.org/en-US/docs/Web/CSS/minmax
https://developer.mozilla.org/en-US/docs/Web/CSS/sign

Reply:
Added `sign`.

`minmax` is not a math function, so I didn't add that in this list.